### PR TITLE
Changed schedule page navigation + style update.

### PR DESCRIPTION
**Pull request by yonimn2000**
Replaced the week selector with previous/next week buttons + updated the header style.
I think this is easier to navigate.

image

### CODE REVIEW

**Comment by yonimn2000** [comment]
See it live: https://hr.yman.dev/#/schedule

File: src/components/SchedulePage.js [hunk 1]
```diff
@@ -79,72 +74,79 @@ export default function SchedulePage() {
...
+            <div  style={{display: "flex"}}>
+              <Button
+                onClick={() => {
+                  let previousMonday = addDays(monday, -7);
+                  setMonday(previousMonday);
+                  refreshSchedule(previousMonday);
+                }}
+                color="primary"
+                variant="contained"
+                style={{textTransform: 'none'}}
```
Comment (sterlingwalker) [inline src/components/SchedulePage.js:131]:
```suggestion
                style={{textTransform: 'none', marginRight: 10}}
```
Just so the buttons don't seem attached. Looks Great!

**Comment by yonimn2000** [comment]
How did you do that cool code suggestion? I did not know you could do that.

**Comment by sterlingwalker** [comment]
> How did you do that cool code suggestion? I did not know you could do that.

@yonimn2000 
It's a small icon that has a +- on it. Looks like this:
image

If you want to comment multiple lines you can hover on the left over the line numbers until your mouse turns blue, click and drag that section for multiline comment.

**Comment by yonimn2000** [comment]
@sterlingwalker Weird. I don't see it :/

image

I found that I can do it manually by using

\```suggestion
[code]
\```

Thanks though!

**Comment by sterlingwalker** [comment]
> @sterlingwalker Weird. I don't see it :/
> 
> image
> 
> I found that I can do it manually by using
> 
> ```suggestion
> [code]
> ```
> 
> Thanks though!

@yonimn2000 Yea I can't do it any more either haha. I think is only for active pull requests

**Comment by yonimn2000** [comment]
You are right! It shows up for an open pull request.

image

**Pull request merged**